Example/Tests/NetworkTests/UploaderTests/UploaderTests.swift

```diff
@@ -27,6 +27,8 @@ import XCTest
 
 class UploaderTests: NetworkBaseTest {
 
+    let allowFolderDecouplingCalls: Bool = false
```

Comment:
Fix to pick from environment variable

Reply:
Hi Alex, if you please change the `allowFolderDecouplingCalls` to be taken from environment variable and not a `Bool`

Reply:
What environment variable specifically?

Reply:
CLD_TEST_FOLDER_DECOUPLING
I would also suggest to look at a function called `shouldSkipTest` in `NetworkBaseTest`